modules/gtk/gtk_mod.c

```diff
@@ -150,6 +156,23 @@ static void menu_on_dial_contact(GtkMenuItem *menuItem, gpointer arg)
 	gtk_mod_connect(mod, uri);
 }
 
+static void menu_on_dial_history(GtkMenuItem *menuItem, gpointer arg)
+{
+	struct gtk_mod *mod = arg;
+	const char *label = gtk_menu_item_get_label(menuItem);
+	char *label_1;
+	char buf[256];
+	char *uri;
+
+	str_ncpy(buf, label, sizeof(buf));
+	label_1 = strchr(buf, '[');
```

Comment:
can strchr return NULL ?

Reply:
while all labels are inserted with an `[`, this theoretically should never be NULL, but since it can return NULL a check against NULL should still be done.